### PR TITLE
Make the MLMD pipeline run information transfer synchronous

### PR DESCRIPTION
**Pull request by stefannica**
## Describe changes
Make the MLMD pipeline run information transfer synchronous

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/zenml/zen_server/deploy/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -35,8 +35,10 @@ spec:
           imagePullPolicy: {{ .Values.zenml.initImage.pullPolicy }}
           command: ['zenml', 'init']
           env:
-            - name: ZENML_DEBUG
-              value: {{ .Values.zenml.debug | quote }}
```
Comment (fa9r) [inline src/zenml/zen_server/deploy/helm/templates/deployment.yaml:39]: shouldn't we set this anyways?

**Review by fa9r — APPROVED** [review]

File: src/zenml/zen_server/deploy/helm/templates/deployment.yaml (hunk 1 above)
Comment (stefannica) [reply src/zenml/zen_server/deploy/helm/templates/deployment.yaml:39]: It's already set in the container image

**Review by AlexejPenner — APPROVED** [review]
LGTM

File: src/zenml/zen_stores/sql_zen_store.py [hunk 2]
```diff
@@ -2766,7 +2766,7 @@ def list_runs(
             if user_name_or_id is not None:
                 user = self._get_user_schema(user_name_or_id, session=session)
                 query = query.where(PipelineRunSchema.user_id == user.id)
-                query = query.order_by(PipelineRunSchema.created)
+            query = query.order_by(PipelineRunSchema.created)
```
Comment (stefannica) [inline src/zenml/zen_stores/sql_zen_store.py:2769]: :rofl: nice catch !

Comment (fa9r) [reply src/zenml/zen_stores/sql_zen_store.py:2769]: 🤦

**Review by schustmi — APPROVED** [review]

**Review by fa9r — APPROVED** [review]

**Pull request merged**